LoopKitUI/View Controllers/ChartsTableViewContainer.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  ChartsTableViewController2.swift
+//  LoopKitUI
+//
+//  Copyright © 2017 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import os.log
+
+public protocol ChartsTableViewControllerDelegate: class {
+    var preferredUnit: HKUnit { get }
+    var healthStore: HKHealthStore { get }
+}
+
+/// Abstract class providing boilerplate setup for chart-based table view controllers
+open class ChartsTableViewContainer: UITableViewController, UIGestureRecognizerDelegate {
```

Comment:
Open to different names for this

Reply:
I would suggest keeping ChartsTableViewController for this, and renaming the inheriting class in Loop, to something like LoopChartsTableViewController